src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c

```diff
@@ -52,98 +54,96 @@ void EnWeatherTag_Destroy(Actor* thisx, GlobalContext* globalCtx) {
             osSyncPrintf("\n\n");
             //"☆☆☆☆☆ Wow wa wa na wa saa ki ha (^o^) ☆☆☆☆☆"
             osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ わわわわー なーがーさーきーはー (^o^) ☆☆☆☆☆ \n" VT_RST);
 
             if (gSaveContext.eventChkInf[4] & 0x400) {
                 Actor_Kill(&this->actor);
             }
-            EnWeatherTag_SetupAction(this, func_80B31924);
+            EnWeatherTag_SetupAction(this, EnWeatherTag_DisabledRainLakeHylia);
             break;
-        case 4:
+        case EN_WEATHER_TAG_TYPE_CLOUDY_DEATH_MOUNTAIN:
             osSyncPrintf("\n\n");
             //"☆☆☆☆☆ Cloudy (._.) Ah Melancholy ☆☆☆☆☆"
             osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ くもり (._.) あーあ 憂鬱 ☆☆☆☆☆ \n" VT_RST);
             if (gSaveContext.eventChkInf[4] & 0x200) {
                 Actor_Kill(&this->actor);
             }
-            EnWeatherTag_SetupAction(this, func_80B317A8);
+            EnWeatherTag_SetupAction(this, EnWeatherTag_DisabledCloudyDeathMountain);
             break;
-        case 5:
+        case EN_WEATHER_TAG_TYPE_THUNDERSTORM_KAKARIKO:
             osSyncPrintf("\n\n");
             //"☆☆☆☆☆ Cloudy Rain Thunder (;O;) Uo Melancholy ☆☆☆☆☆"
             osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ くもり雨雷 (;O;) うおお 憂鬱 ☆☆☆☆☆ \n" VT_RST);
 
-            temp_v0 = gSaveContext.eventChkInf[4];
-            if ((((temp_v0 & 0x100) == 0) || ((temp_v0 & 0x200) == 0) || ((temp_v0 & 0x400) == 0)) ||
-                CHECK_QUEST_ITEM(QUEST_MEDALLION_SHADOW)) {
+            if (!(gSaveContext.eventChkInf[4] & 0x100) || !(gSaveContext.eventChkInf[4] & 0x200) ||
+                !(gSaveContext.eventChkInf[4] & 0x400) || CHECK_QUEST_ITEM(QUEST_MEDALLION_SHADOW)) {
                 Actor_Kill(&this->actor);
             }
-            EnWeatherTag_SetupAction(this, func_80B31A10);
+            EnWeatherTag_SetupAction(this, EnWeatherTag_DisabledCloudyRainThunderKakariko);
             break;
-        case 6:
+        case EN_WEATHER_TAG_TYPE_SANDSTORM_INTENSITY:
             osSyncPrintf("\n\n");
             //"☆☆☆☆☆ The desert becomes thicker ☆☆☆☆☆"
             osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ 砂漠が濃くなります ☆☆☆☆☆ \n" VT_RST);
-            EnWeatherTag_SetupAction(this, func_80B31B24);
+            EnWeatherTag_SetupAction(this, EnWeatherTag_SetSandstormIntensity);
             break;
-        case 7:
+        case EN_WEATHER_TAG_TYPE_THUNDERSTORM_GRAVEYARD:
             osSyncPrintf("\n\n");
             //"☆☆☆☆☆ Wow wa wa na wa saa ki ha (^o^) ☆☆☆☆☆"
             osSyncPrintf(VT_FGCOL(YELLOW) "☆☆☆☆☆ わわわわー なーがーさーきーはー (^o^) ☆☆☆☆☆ \n" VT_RST);
 
-            EnWeatherTag_SetupAction(this, func_80B31C20);
+            EnWeatherTag_SetupAction(this, EnWeatherTag_DisabledRainThunder);
             break;
     }
 }
 
-u8 func_80B312C8(EnWeatherTag* this, GlobalContext* globalCtx, u8 arg2, u8 arg3, u8 arg4, u8 arg5, u16 arg6, u8 arg7) {
+u8 CheckEnableWeatherEffect(EnWeatherTag* this, GlobalContext* globalCtx, u8 arg2, u8 arg3, u8 arg4, u8 arg5, u16 arg6,
```

Comment:
This function name doesn't fit our function naming scheme (ie. `SystemName_Function`)